pkg/api/types.go

```diff
@@ -1479,6 +1479,18 @@ type NodeStatus struct {
 	DaemonEndpoints NodeDaemonEndpoints `json:"daemonEndpoints,omitempty"`
 	// Set of ids/uuids to uniquely identify the node.
 	NodeInfo NodeSystemInfo `json:"nodeInfo,omitempty"`
+	// List of container images on this node
+	Images []ContainerImage `json:"images",omitempty`
+}
+
+// Describe a container image
+type ContainerImage struct {
+	// ID of the image.
+	ID string
```

Comment:
Do we need to expose the image ID? We can't pull the image by ID, and scheduler would have no knowledge what the image ID from an ImageSpec would be.